common/on-chip-calib.cpp

```diff
@@ -400,17 +400,6 @@ namespace rs2
         bool frame_arrived = false;
         try
         {
-            if (_sub->s->supports(RS2_OPTION_EMITTER_ENABLED))
```

Comment:
This could lead to issues no?
It affect all calibrations no?

Reply:
Yes, it's a bug that affects all calibrations.
I moved the code to another place that affects all calibrations but is called only once.